README.md

```diff
@@ -331,7 +331,9 @@ $ kubectl delete -f pvc.yaml
 persistentvolumeclaim "csi-zfspv" deleted
 ```
 
+## Roadmap
```

Comment:
n_: can we push this right next to the status, at the top of this file.

Reply:
done.